mysql-test/include/log_encrypt_6.inc

```diff
@@ -11,6 +11,7 @@ call mtr.add_suppression("Error while loading keyring content. The keyring might
 call mtr.add_suppression("\\[Error\\] InnoDB: Encryption can't find masterkey, please check the keyring plugin is loaded.");
 call mtr.add_suppression("\\[ERROR\\] InnoDB: Failed to find tablespace for table `\.\.*`\.`\.\.*` in the cache.");
 call mtr.add_suppression("\\[ERROR\\] InnoDB: If you are installing InnoDB, remember that you must create directories yourself, InnoDB does not create them.");
+call mtr.add_suppression ("Redo log key generation failed");
```

Comment:
This is supposed to be removed? (from our discussion on 5.7 PR)

Reply:
Updated: the 8.0 test remained unstable even after the fix I've used in 5.7. I updated both branches differently: the background thread should never try to generate a new key, it only checks for a newer, but existing key.